components/automate-cli/cmd/chef-automate/backup.go

```diff
@@ -28,6 +28,15 @@ import (
 	"github.com/chef/automate/lib/io/fileutils"
 )
 
+const restoreRecovery = `Check the logs (journalctl -f -u chef-automate) for errors related
+to the failed restore.
+
+Before retrying the restore process, remove the file %s.
+
+If you are restoring over an existing Chef Automate installation, also ensure that
+installation is in a known state such that either all services are up (chef-automate restart-services)
+or services are down (chef-automate stop) before retrying the restore.`
```

Comment:
```suggestion
Before restoring over an existing Chef Automate installation all services need to be either running or down. The restore will not work if it is run while services are in an unknown state. To restart services use chef-automate restart-services. To stop services use chef-automate stop
```

idk if thats any better?

Reply:
@susanev Under normal circumstances, restore will work even if some of the services are down. In the case of the issue that motivated adding the recovery message, the restart was necessary because of weird state further down the stack. I'll think about if/how to rephrase.